### PR TITLE
Fixed 'Create Account' mnemonic

### PR DESCRIPTION
**Pull request by ozalexo**

### CODE REVIEW

File: packages/login-ui/components/ConfirmMnemonic/ConfirmMnemonic.js [hunk 1]
```diff
@@ -12,84 +12,41 @@ import Button from 'components/common/ui/Button/Button'
...
-    this.setState(
-      { confirmPhrase: this.state.confirmPhrase.slice(0, -1) },
-      () => change('mnemonic', this.getCurrentMnemonic())
-    )
-  }
+  getWordsButtons = () =>
+    this.props.currentWordsArray
+      .map((item) => {
+        const wordSelected = this.props.confirmPhrase.includes(item.word)
+        const keyIndex = [item.word, item.index].join('')
```
Comment (wAISw) [inline packages/login-ui/components/ConfirmMnemonic/ConfirmMnemonic.js]: `${name}${key}` use it

File: packages/login-ui/components/ConfirmMnemonic/ConfirmMnemonicContainer.js [hunk 2]
```diff
@@ -36,23 +71,49 @@ export default class ConfirmMnemonicContainer extends Component {
...
+  handleClearLastWord = () => {
+    this.setState(
+      { confirmPhrase: this.state.confirmPhrase.slice(0, -1) },
+      () => this.refreshCurrentMnemonicFormInput()
+    )
+  }
+
+  refreshCurrentMnemonicFormInput = () => {
+    const confPhrase = this.state.confirmPhrase
+      .map((item) => item)
```
Comment (wAISw) [inline packages/login-ui/components/ConfirmMnemonic/ConfirmMnemonicContainer.js]: useless ?!

File: packages/login-ui/components/ConfirmMnemonic/ConfirmMnemonic.js [hunk 3]
```diff
@@ -12,84 +12,41 @@ import Button from 'components/common/ui/Button/Button'
...
+            isWordSelected={wordSelected}
+            mnemonicWord={item.word}
+            onClick={this.props.onClickWord}
+          />
+        )
+      })
+
+  getCurrentMnemonic = () =>
+    this.props.confirmPhrase
+      .map((item) => item)
```
Comment (wAISw) [inline packages/login-ui/components/ConfirmMnemonic/ConfirmMnemonic.js]: useless

File: packages/login-ui/components/ConfirmMnemonic/ConfirmMnemonic.js (hunk 1 above)
Comment (e-maximus) [reply packages/login-ui/components/ConfirmMnemonic/ConfirmMnemonic.js]: +

Comment (ozalexo) [reply packages/login-ui/components/ConfirmMnemonic/ConfirmMnemonic.js]: I do not agree. ${} is a template syntax, for JSX etc.

**Pull request merged**